datasets/ro/fiu_declarations/ro_fiu_declarations.yml

```diff
@@ -0,0 +1,37 @@
+title: Romania FIU Public Officials
+url: https://www.onpcsb.ro/en/a/101/declarations-of-assets-and-interests
+entry_point: crawler.py
+prefix: ro-fiu
+load_db_uri: ${OPENSANCTIONS_DATABASE_URI}
+coverage:
+  frequency: weekly
+  start: 2024-12-20
+summary: >
+  List of the public officials in the Financial Intelligence Unit of Romania (FIU).
+description: |
+  The dataset includes details of public officials in the Financial Intelligence Unit 
+  of Romania required to file asset declarations. These declarations are part of measures 
+  aimed at ensuring transparency and preventing illicit financial activities.
+publisher:
+  name: Financial Intelligence Unit of Romania
+  acronym: FIU
+  official: true
+  description: |
+    The National Office for Preventing and Combating Money Laundering is the Financial 
+    Intelligence Unit of Romania (FIU) of administrative type, with a leading role in 
+    the development, coordination and implementation of the system for combating money 
+    laundering and terrorist financing.
+    [Source: Official Website](https://www.onpcsb.ro/en/a/115/general-framework)
```

Comment:
if we're quoting, let's prefix the rows with `>` to make a markdown quote. But it might be nice to write this ourselves to make it less "marketing speak" (e.g. "leading role")